gdp/column/gdp_column.py

```diff
@@ -258,114 +230,132 @@ def one_reflux(m):
     def one_boilup(m):
         return exactly(1, m.YB)
 
+    # Defining logical constraint for YP Boolean variable
     @m.LogicalConstraint(m.conditional_trays)
     def YP_or_notYP(m, n):
         return m.YP[n].equivalent_to(land(lor(m.YR[j] for j in range(n, m.max_trays)), lor(land(~m.YB[j] for j in range(n, m.max_trays)), m.YB[n])))
 
-    # Associate Boolean variables with with disjunctions
+    # Associating YP Boolean variable with tray activation
     for n in m.conditional_trays:
         m.YP[n].associate_binary_var(m.tray[n].indicator_var)
 
-    # Fix feed conditions
-    m.feed['benzene'].fix(50)
-    m.feed['toluene'].fix(50)
-    m.T_feed.fix(368)
-    m.feed_vap_frac.fix(0.40395)
-    m.P.fix(1.01)
-    # Fix to be total condenser
-    m.partial_cond.deactivate()
-    m.total_cond.indicator_var.fix(1)
-
-    # Fix auxiliary Boolean variables denoting logical position of boilup and reflux
+    # Fixing feed conditions
+    m.feed['benzene'].fix(50)  # Fixing benzene flow in the feed at 50 mol/s
```

Comment:
Fix Units formatting

Reply:
[mol/s] although check if these are the actual units we are using

Reply:
Changed into [kmol/s], Adjust the error and added square brackets surrounding the units.